heap/max_heap.py

```diff
@@ -30,13 +30,22 @@ def _bubble_up(self, index):
 
 
   def _sift_down(self, index):
-    left_child = (index * 2) + 1
-    right_child = (index * 2) + 2
-    parent = (index - 1) // 2
     # if the left node is greater than the right node, check left node against parent node. If child node has higher value swap them
-    if self.storage[left_child] > self.storage[right_child]:
-      if self.storage[left_child] > self.storage[parent]:
-        self.storage[left_child], self.storage[parent] = self.storage[parent], self.storage[left_child]
-    elif self.storage[right_child] > self.storage[left_child]:
-      if self.storage[right_child] > self.storage[parent]:
-        self.storage[right_child], self.storage[parent] = self.storage[parent], self.storage[right_child]
+    while (index * 2) + 1 <= len(self.storage) -1:
+      if (index * 2) + 2 > len(self.storage) -1:
+        big_num = (index * 2) + 1
```

Comment:
lines 36 and 38 are the same. can you maybe join lines 35 and 37 with an `or` statement?